### PR TITLE
[SYSDM][USRMGR] French translation update (CORE-16596)

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- Completing French translation for User and Hardware profiles

JIRA issue: [CORE-16596](https://jira.reactos.org/browse/CORE-16596)

## Proposed changes

VirtualBox_ReactOS_12_05_2020_22_27_30
userprof
VirtualBox_ReactOS_12_05_2020_22_26_09

### CODE REVIEW

File: dll/cpl/sysdm/lang/fr-FR.rc [hunk 1]
```diff
@@ -167,29 +167,29 @@ END
...
-CAPTION "Copy To"
+CAPTION "Copier vers"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Copy &profile to", -1, 8, 9, 185, 52
+    GROUPBOX "Copier le &profil vers", -1, 8, 9, 185, 52
     EDITTEXT IDC_USERPROFILE_COPY_PATH, 15, 23, 167, 12, ES_AUTOHSCROLL
-    PUSHBUTTON "&Browse", IDC_USERPROFILE_COPY_BROWSE, 15, 41, 50, 14
-    GROUPBOX "Permitted to use", -1, 8, 77, 185, 48
+    PUSHBUTTON "&PArcourir", IDC_USERPROFILE_COPY_BROWSE, 15, 41, 50, 14
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/fr-FR.rc:187]:
```suggestion
    PUSHBUTTON "&Parcourir", IDC_USERPROFILE_COPY_BROWSE, 15, 41, 50, 14
```

File: dll/cpl/sysdm/lang/fr-FR.rc [hunk 2]
```diff
@@ -323,13 +323,13 @@ BEGIN
...
-    IDS_USERPROFILE_MANDATORY "Mandatory"
-    IDS_USERPROFILE_CONFIRM_DELETE "Are you sure you want to delete %s's profile?"
-    IDS_USERPROFILE_CONFIRM_DELETE_TITLE "Confirm Delete"
-    IDS_USERPROFILE_ACCOUNT_UNKNOWN "Account Unknown"
-    IDS_USERPROFILE_ACCOUNT_DELETED "Account Deleted"
-    IDS_USERPROFILE_TYPE_TEXT "When %s logs onto this computer, should the operating system use the roaming profile or just the locally cached copy of the roaming profile."
+    IDS_USERPROFILE_ROAMING "Itinérant"
+    IDS_USERPROFILE_MANDATORY "Obligatoire"
+    IDS_USERPROFILE_CONFIRM_DELETE "Voulez-vous supprimer le profil de %s ?"
+    IDS_USERPROFILE_CONFIRM_DELETE_TITLE "Confirmer suppression"
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/fr-FR.rc:329]:
```suggestion
    IDS_USERPROFILE_CONFIRM_DELETE_TITLE "Confirmer la suppression"
```

File: dll/cpl/usrmgr/lang/fr-FR.rc [hunk 3]
```diff
@@ -9,11 +9,11 @@ FONT 8, "MS Shell Dlg"
...
             LVS_SHOWSELALWAYS | LVS_SORTASCENDING | WS_BORDER | WS_TABSTOP, 7, 7, 238, 85, WS_EX_CLIENTEDGE
-    PUSHBUTTON "A&dd...", IDC_USERS_ADD, 87, 98, 50, 14
-    PUSHBUTTON "&Remove", IDC_USERS_REMOVE, 141, 98, 50, 14
-    PUSHBUTTON "Pr&operties", IDC_USERS_PROPERTIES, 195, 98, 50, 14
-    AUTOCHECKBOX "Require user name and password on system startup.", IDC_USERS_STARTUP_REQUIRE, 10, 118, 200, 14, WS_DISABLED
-    PUSHBUTTON "Reset Password", IDC_USERS_RESET, 165, 200, 80, 14, WS_DISABLED
+    PUSHBUTTON "A&jouter...", IDC_USERS_ADD, 87, 98, 50, 14
+    PUSHBUTTON "&Retirer", IDC_USERS_REMOVE, 141, 98, 50, 14
+    PUSHBUTTON "Pr&opriétés", IDC_USERS_PROPERTIES, 195, 98, 50, 14
+    AUTOCHECKBOX "Nécessite un identifiant et mot de passe au démarrage système.", IDC_USERS_STARTUP_REQUIRE, 10, 118, 200, 14, WS_DISABLED
```
Comment (HBelusca) [inline dll/cpl/usrmgr/lang/fr-FR.rc:15]:
```suggestion
    AUTOCHECKBOX "Nécessite un identifiant et mot de passe au démarrage du système.", IDC_USERS_STARTUP_REQUIRE, 10, 118, 200, 14, WS_DISABLED
```

Comment (HBelusca) [reply dll/cpl/usrmgr/lang/fr-FR.rc:15]: (optional, don't apply if the checkbox width cannot be increased without changing the whole layout of the surrounding elements)

File: dll/cpl/usrmgr/lang/fr-FR.rc [hunk 4]
```diff
@@ -9,11 +9,11 @@ FONT 8, "MS Shell Dlg"
...
+    PUSHBUTTON "Réinitialiser le mot de passe", IDC_USERS_RESET, 145, 200, 100, 14, WS_DISABLED
```
Comment (HBelusca) [inline dll/cpl/usrmgr/lang/fr-FR.rc:16]:
```suggestion
    PUSHBUTTON "Réinitialiser le &mot de passe", IDC_USERS_RESET, 145, 200, 100, 14, WS_DISABLED
```

File: dll/cpl/sysdm/lang/fr-FR.rc [hunk 5]
```diff
@@ -102,22 +102,22 @@ END
...
+    LTEXT "Numéro de série :", IDC_STATIC, 10, 55, 93, 8
+    LTEXT "", IDC_HWPPROP_SERIAL, 103, 55, 135, 8
     GROUPBOX "", IDC_STATIC, 10, 69, 192, 61
-    AUTOCHECKBOX "This is a &portable computer", IDC_HWPPROP_PORTABLE, 17, 68, 115, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "The docking &state is unknown", IDC_HWPPROP_UNKNOWN, 22, 85, 160, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "The computer is &docked", IDC_HWPPROP_DOCKED, 22, 97, 160, 10
-    AUTORADIOBUTTON "The computer is &undocked", IDC_HWPPROP_UNDOCKED, 22, 109, 160, 10
-    GROUPBOX "Hardware profiles selection", IDC_STATIC, 10, 138, 192, 42
-    AUTOCHECKBOX "Always &include this profile as an option when ReactOS starts", IDC_HWPPROP_INCLUDE, 17, 150, 164, 20, WS_GROUP | WS_TABSTOP | BS_MULTILINE
+    AUTOCHECKBOX "Ceci est un ordinteur portable", IDC_HWPPROP_PORTABLE, 17, 68, 115, 10, WS_GROUP | WS_TABSTOP
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/fr-FR.rc:115]:
```suggestion
    AUTOCHECKBOX "Ceci est un ordinteur &portable", IDC_HWPPROP_PORTABLE, 17, 68, 115, 10, WS_GROUP | WS_TABSTOP
```

File: dll/cpl/sysdm/lang/fr-FR.rc [hunk 6]
```diff
@@ -102,22 +102,22 @@ END
...
+    AUTORADIOBUTTON "L'état de la station d'accueil est inconnu", IDC_HWPPROP_UNKNOWN, 22, 85, 160, 10, WS_GROUP | WS_TABSTOP
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/fr-FR.rc:116]:
```suggestion
    AUTORADIOBUTTON "L'état de la &station d'accueil est inconnu", IDC_HWPPROP_UNKNOWN, 22, 85, 160, 10, WS_GROUP | WS_TABSTOP
```

File: dll/cpl/sysdm/lang/fr-FR.rc [hunk 7]
```diff
@@ -102,22 +102,22 @@ END
...
+    AUTORADIOBUTTON "L'ordinateur est connecté à sa station d'accueil", IDC_HWPPROP_DOCKED, 22, 97, 160, 10
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/fr-FR.rc:117]:
```suggestion
    AUTORADIOBUTTON "L'ordinateur est &connecté à sa station d'accueil", IDC_HWPPROP_DOCKED, 22, 97, 160, 10
```

File: dll/cpl/sysdm/lang/fr-FR.rc [hunk 8]
```diff
@@ -102,22 +102,22 @@ END
...
+    AUTORADIOBUTTON "L'ordinateur est retiré de sa station d'accueil", IDC_HWPPROP_UNDOCKED, 22, 109, 160, 10
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/fr-FR.rc:118]:
```suggestion
    AUTORADIOBUTTON "L'ordinateur est &retiré de sa station d'accueil", IDC_HWPPROP_UNDOCKED, 22, 109, 160, 10
```

File: dll/cpl/sysdm/lang/fr-FR.rc [hunk 9]
```diff
@@ -102,22 +102,22 @@ END
...
+    GROUPBOX "Sélection de profils matériels", IDC_STATIC, 10, 138, 192, 42
+    AUTOCHECKBOX "Toujours inclure ce profil en tant qu'option lors du démarrage de ReactOS", IDC_HWPPROP_INCLUDE, 17, 150, 164, 20, WS_GROUP | WS_TABSTOP | BS_MULTILINE
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/fr-FR.rc:120]:
```suggestion
    AUTOCHECKBOX "Toujours &inclure ce profil en tant qu'option lors du démarrage de ReactOS", IDC_HWPPROP_INCLUDE, 17, 150, 164, 20, WS_GROUP | WS_TABSTOP | BS_MULTILINE
```

File: dll/cpl/sysdm/lang/fr-FR.rc (hunk 1 above)
Comment (KyleKatarn31) [reply dll/cpl/sysdm/lang/fr-FR.rc:187]: ok

File: dll/cpl/sysdm/lang/fr-FR.rc (hunk 2 above)
Comment (KyleKatarn31) [reply dll/cpl/sysdm/lang/fr-FR.rc:329]: ok

File: dll/cpl/usrmgr/lang/fr-FR.rc (hunk 3 above)
Comment (KyleKatarn31) [reply dll/cpl/usrmgr/lang/fr-FR.rc:15]: Yes, checkbox extended accordingly

File: dll/cpl/usrmgr/lang/fr-FR.rc (hunk 4 above)
Comment (KyleKatarn31) [reply dll/cpl/usrmgr/lang/fr-FR.rc:16]: ok

File: dll/cpl/sysdm/lang/fr-FR.rc (hunk 5 above)
Comment (KyleKatarn31) [reply dll/cpl/sysdm/lang/fr-FR.rc:115]: ok

File: dll/cpl/sysdm/lang/fr-FR.rc (hunk 8 above)
Comment (KyleKatarn31) [reply dll/cpl/sysdm/lang/fr-FR.rc:118]: ok

File: dll/cpl/sysdm/lang/fr-FR.rc (hunk 7 above)
Comment (KyleKatarn31) [reply dll/cpl/sysdm/lang/fr-FR.rc:117]: ok

File: dll/cpl/sysdm/lang/fr-FR.rc (hunk 6 above)
Comment (KyleKatarn31) [reply dll/cpl/sysdm/lang/fr-FR.rc:116]: ok

File: dll/cpl/sysdm/lang/fr-FR.rc (hunk 9 above)
Comment (KyleKatarn31) [reply dll/cpl/sysdm/lang/fr-FR.rc:120]: ok

**Comment by KyleKatarn31** [comment]
Thank you @HBelusca : All comments taken into account.

**Comment by KyleKatarn31** [comment]
Thanks !

**Pull request merged**